tests/wrappers/huggingface/token_classification_classifier_test.py

```diff
@@ -0,0 +1,250 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for TestTokenClassification processor.
+"""
+import unittest
+
+from forte.data.data_pack import DataPack
+from forte.pipeline import Pipeline
+from forte.data.readers import StringReader
+from forte.nltk import NLTKSentenceSegmenter
+from forte.huggingface import TokenClassification
+
+
+class TestTokenClassification(unittest.TestCase):
+    def test_huggingface_ner_bio_classification(self):
+        nlp = Pipeline[DataPack](enforce_consistency=True)
+        nlp.set_reader(StringReader())
+        nlp.add(NLTKSentenceSegmenter())
```

Comment:
We can reduce the dependency to NLTK by using this sentence splitter based on periods (https://github.com/asyml/forte/blob/master/forte/processors/misc/simple_processors.py#L26)

Reply:
Using this processor will lead a error on pipeline:
forte.common.exception.ExpectedRecordNotFound: The record type ft.onto.base_ontology.Sentence is not found in meta of the prediction datapack

So I continued with NLTK.

Reply:
Remove the `enforce_consistency` argument in the pipeline, which will result in the pipeline not checking the expected types.

btw, next time, please do not resolve the conversation so that I can see your replies (otherwise they will be collapsed).__